go.mod

```diff
@@ -13,6 +13,7 @@ require (
 	github.com/gabriel-vasile/mimetype v1.4.0
 	github.com/gin-gonic/gin v1.7.7
 	github.com/gogo/protobuf v1.3.3
+	github.com/golang-jwt/jwt v3.2.2+incompatible
```

Comment:
Is there a specific reason to be using 3.x instead of 4.x here? Just curious, I didn't really go deeply into the differences between the versions.